src/components/StartupForm/StartupForm.tsx

```diff
@@ -182,7 +182,7 @@ export function StartupForm(props: StartupFormProps) {
                 if (resp.success) {
                     setAlertMessage({
                         title: `Mise à jour effectuée`,
-                        message: `La mise à jour a bien été effectuée`,
+                        message: `La mise à jour a bien été effectuée. Elle sera visible en ligne d'ici 24 heures.`,
```

Comment:
```suggestion
                        message: `La modification sera visible en ligne d'ici 24 heures.`,
```
(pas besoin de répéter que ça a été effectué)